article/sections/_9_conclusion.tex

```diff
@@ -0,0 +1,42 @@
+\section{Conclusion}
+\label{sec:conclusion}
+
+Travel demand problems aim at forecasting the impact of proposed project and policies. These problems are causal in nature. 
+To our surprise, there has been very little mention of causal inference in the field of travel demand modeling.
+\citet{brathwaite_2018_causal} have documented this disconnect and presented an initial framework for addressing it.
+
+In this chapter, we build up on \citet{brathwaite_2018_causal} and highlight the importance of using causal graphs and 
+causal inference methods in transportation demand modeling efforts.
```

Comment:
I suggest removing the citation as it seemed visually repetitive with the two citations only being separated by a sentence.

```suggestion
In this chapter, we built upon their framework.
We highlighted the importance of using causal graphs and
causal inference methods in transportation demand modeling efforts and in choice modeling more generally.
```